pkg/webauthnutil/webauthnutil.go

```diff
@@ -2,14 +2,30 @@
 package webauthnutil
 
 import (
+	"net"
 	"net/http"
+	"strings"
 
-	"github.com/pomerium/pomerium/internal/urlutil"
 	"github.com/pomerium/pomerium/pkg/grpc/databroker"
 	"github.com/pomerium/webauthn"
 )
 
 // GetRelyingParty gets a RelyingParty for the given request and databroker client.
 func GetRelyingParty(r *http.Request, client databroker.DataBrokerServiceClient) *webauthn.RelyingParty {
-	return webauthn.NewRelyingParty(urlutil.GetOrigin(r), NewCredentialStorage(client))
+	return webauthn.NewRelyingParty(
+		"https://"+GetEffectiveDomain(r),
+		NewCredentialStorage(client),
+	)
+}
+
+// GetEffectiveDomain returns the effective domain for an HTTP request.
+func GetEffectiveDomain(r *http.Request) string {
```

Comment:
Maybe possible to use https://pkg.go.dev/golang.org/x/net/publicsuffix#EffectiveTLDPlusOne instead?